adapters/huaweiads/huaweiads.go

```diff
@@ -50,25 +50,37 @@ const (
 	videoWithPicturesText  int32 = 11
 )
 
+// interaction type
+const (
+	appPromotion int32 = 3
+	appDownload  int32 = 2
```

Comment:
`appDownload` isn't used anywhere. Can you remove this?

Reply:
deleted